synapse/util/ratelimitutils.py

```diff
@@ -35,6 +37,11 @@
 logger = logging.getLogger(__name__)
 
 
+rate_limit_sleep_counter = Counter("synapse_rate_limit_sleep", "", ["host"])
+
+rate_limit_reject_counter = Counter("synapse_rate_limit_reject", "", ["host"])
```

Comment:
I'd vote for not adding `"host"` here, given that is an unbounded variable. If we add some logging for when we're sleeping hosts then the metric would allow us to see that we've triggered it and the logging which hosts are affected?

Reply:
Makes sense 👍. Was focused about what was in the doc, [metrics per-host](https://docs.google.com/document/d/1lvUoVfYUiy6UaHB6Rb4HicjaJAU40-APue9Q4vzuW3c/edit#bookmark=id.zjko88lr25j).

Currently, the logs are mostly `debug` here so we'll have to turn them on when we want to see.

---

Affected host count per time-period might be interesting to see so we can differentiate one really noisy homeserver from a general ratelimit tuning problem across the federation. I guess we would add a gauge for this. I can follow-up in another PR for this.